stablehlo/reference/Interpreter.cpp

```diff
@@ -90,16 +90,16 @@ llvm::Expected<SmallVector<Tensor>> eval(func::FuncOp func,
       Tensor runtimeResult = eval(constantOp, constantOp.value());
       populateResults({runtimeResult});
     } else if (auto cosineOp = dyn_cast<CosineOp>(op)) {
-      Tensor runtimeOperand = fetchOperand(cosineOp.operand());
-      Tensor runtimeResult = eval(cosineOp, runtimeOperand);
+      Tensor runtimeOpr = fetchOperand(cosineOp.operand());
```

Comment:
I haven't seen "operand" abbreviated to "opr", so how about we keep the existing name?

Reply:
Sure, then since CeilOp and FloorOp uses `runtimeOpr`, I will change them to `runtimeOperand`